### PR TITLE
CAN Signal Single Point IO example

### PR DESCRIPTION
**Pull request by jashnani**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nixnet-python/blob/master/CONTRIBUTING.rst).
- [ ] New tests have been created for any new features or regression tests for bugfixes.
- [ ] `tox` successfully runs, including unit tests and style checks (see [CONTRIBUTING.md](https://github.com/ni/nixnet-python/blob/master/CONTRIBUTING.rst)).

TODO: Check the above boxes with an 'x' to indicate what steps you have taken in preparing this Pull Request.

### What does this Pull Request accomplish?

Adds CAN Signal Single Point IO example and enhances the OOP API layer as needed.

### Why should this Pull Request be merged?

This is a stepping stone to fleshing out the API.

### What testing has been done?

None

### CODE REVIEW

File: nixnet_examples/can_signal_single_point_io.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                print('Sent signal values: %s' % value_buffer)
+
+                # Wait 1 s and then read the received values.
+                # They should be the same as the ones sent.
+                time.sleep(1)
+
+                num_signals = len(value_buffer)
+                signals = input_session.read_signal_single_point(num_signals)
+                for timestamp, value in signals:
+                    print('Received signal value: %s' % value)
```
Comment (epage) [inline nixnet_examples/can_signal_single_point_io.py]: Let's print the timestamps so we make sure they are right.

To make it fancier
```python
datetime.fromtimestamp(timestamp / 1e-9)
```
(that should convert it from nanoseconds to a datetime)

Comment (jashnani) [inline nixnet_examples/can_signal_single_point_io.py]: Fixed

Comment (epage) [inline nixnet_examples/can_signal_single_point_io.py]: We both missed my math mistake.  We either multiple by 1e-9 or divide by 1e9 to turn nanoseconds into fractional seconds.

Comment (jashnani) [inline nixnet_examples/can_signal_single_point_io.py]: Good catch! We are now dividing by 1e9

**Pull request merged**